src/ast/mod.rs

```diff
@@ -91,14 +100,15 @@ pub enum Expression {
     Int(u32),
     Str(String),
     Bool(bool),
+    Selff,
```

Comment:
What's this?

Reply:
The "self" keyword. `Self` is a reserved keyword in rust.

Reply:
Maybe you should add a comment for it so people won't get confused?

Reply:
Done👍